mango_time_series/mango_time_series/features/date_utils.py

```diff
@@ -0,0 +1,119 @@
+from datetime import datetime, timedelta
+
+import polars as pl
+
+from mango_time_series.features.calendar_features import get_calendar
+
+
+def get_holidays_df(steps_back: int, steps_forward: int) -> pl.DataFrame:
+    """
+    Get national holidays dataframe with window bounds.
+
+    :param steps_back: int, number of days to go back from the holiday.
+    :param steps_forward: int, number of days to go forward
+    from the holiday.
+    :return: pd.DataFrame, holidays dataframe with window
+    """
+    start_year = 2014
+    all_holidays = get_calendar(
+        "ES",
```

Comment:
Could this me an argument? Maybe by default it is ES

Reply:
Does it make sense that we can also choose if we want a pandas dataframe or a polars back?